### PR TITLE
More redirect updates

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
More fixes to redirects.

### Issues Resolved
n/a

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by Naarcha-AWS — APPROVED** [review]

File: _upgrade-to/upgrade-to.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ If you are migrating an Open Distro for Elasticsearch cluster, we recommend firs
 
    1. Port your settings from `elasticsearch.yml` to `opensearch.yml`. Most settings use the same names. At a minimum, specify `cluster.name`, `node.name`, `discovery.seed_hosts`, and `cluster.initial_cluster_manager_nodes`.
 
-   1. (Optional) If you're actively connecting to the cluster with legacy clients that check for a particular version number, such as Logstash OSS, add a [compatibility setting]({{site.url}}{{site.baseurl}}/clients/agents-and-ingestion-tools/) to `opensearch.yml`:
+   1. (Optional) If you're actively connecting to the cluster with legacy clients that check for a particular version number, such as Logstash OSS, add a [compatibility setting]({{site.url}}{{site.baseurl}}/https://opensearch.org/docs/2.7/tools/index/#compatibility-matrices/) to `opensearch.yml`:
```
Comment (cwillum) [inline _upgrade-to/upgrade-to.md:147]: This link doesn't work for me. I built local to try it. (But the 404 page works!)

**Review by cwillum — APPROVED** [review]
Just one comment at the bottom having to do with a link.

File: _upgrade-to/upgrade-to.md (hunk 1 above)
Comment (hdhalter) [reply _upgrade-to/upgrade-to.md:147]: It works for me locally. @Naarcha-AWS or @kolchfa-aws - do you mind testing this link? Thank you!

Comment (kolchfa-aws) [reply _upgrade-to/upgrade-to.md:147]: Remove the first prefix from the link (starting with https)

File: _tools/index.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ This section provides documentation for OpenSearch-supported tools, including:
 - [OpenSearch CLI](#opensearch-cli)
 - [OpenSearch Kubernetes operator](#opensearch-kubernetes-operator)
 
+For information about Data Prepper, the server-side data collector for filtering, enriching, transforming, normalizing, and aggregating data for downstream analytics and visualization, see [Data Prepper]({{site.url}}{{site.baseurl}}/ddata-prepper/index/).
```
Comment (kolchfa-aws) [inline _tools/index.md:19]: Extra d in data prepper

**Pull request merged**